backend/src/main/java/WebApi.java

```diff
@@ -7,9 +10,51 @@
     private static WebInterface webInterface = new WebInterfaceImpl();
 
     public static void main(String[] args) {
+        WebApi.enableCORS("*","*","*");
+
+        Gson gson = new Gson();
         get("/hello", (req, res) -> "Hello World");
 
-        get("/stl/:stlID", (req, res) -> webInterface.getAllPrintsForStl(req.params("stlID")));
+        // Customers
+        get("/customers", (request, response) -> webInterface.getAllCustomers(), gson::toJson);
+        get("/customers/:customerID,", ((request, response) -> webInterface.getCustomer(request.params(":customerID"))), gson::toJson);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "customerID" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)